### PR TITLE
allow listing clusters and add help text

### PR DESCRIPTION
**Pull request by hjacobs**
Another usability improvement:

* allow listing clusters with `zkubectl list-clusters`
* show help text with `zkubectl help` or `zkubectl -h`
* add informational output for `zkubectl login` (most users expect some output/feedback..)

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -127,3 +127,7 @@ def test_looks_like_url():
     assert zalando_kubectl.main.looks_like_url('https://localhost')
     assert zalando_kubectl.main.looks_like_url('http://localhost')
     assert zalando_kubectl.main.looks_like_url('foo.example.org')
+
+
+def test_print_help():
```
Comment (prayerslayer) [inline tests/test_cli.py]: 👍

**Comment by prayerslayer** [comment]
👍

File: tests/test_cli.py (hunk 1 above)
Comment (hjacobs) [inline tests/test_cli.py]: Yeah, testing done by professionals :smirk:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**